demo-shell/src/app/services/app-notifications.service.ts

```diff
@@ -43,9 +44,16 @@ export class AppNotificationsService {
         private notificationService: NotificationService,
         private translateService: TranslateService,
         private identityUserService: IdentityUserService,
-        private alfrescoApiService: AlfrescoApiService
+        private alfrescoApiService: AlfrescoApiService,
+        private authenticationService: AuthenticationService
     ) {
-        this.alfrescoApiService.alfrescoApiInitialized.subscribe(() => {
+        merge(this.alfrescoApiService.alfrescoApiInitialized, this.authenticationService.onLogin)
```

Comment:
this.authenticationService.onLogin seems not be triggered when is in SSO probably we need to fix something there as well.

Reply:
I've checked with SSO and it works fine, let me have another look